docs/PERFORMANCE_USAGE.md

```
@@ -0,0 +1,228 @@
+trends = manager.analyze_trends(days=7)
+
+# Check against thresholds
+violations = monitor.check_thresholds({
+    "duration_ms": 100,
+    "cpu_percent": 50
+})
+```
+
+## Best Practices
+
+### 1. Operation Monitoring
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)